ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/cli/Cli.scala

```diff
@@ -87,7 +87,7 @@ object Cli {
     opt[String](name = "scenario")
       .action((x, c) => c.copy(scenario = Some(x)))
       .text(
-        "If set, the sandbox will execute the given scenario on startup and store all the contracts created by it. " +
+        "DEPRECATED! If set, the sandbox will execute the given scenario on startup and store all the contracts created by it. " +
```

Comment:
This seems quite aggressive. How about
```suggestion
        "If set, the sandbox will execute the given scenario on startup and store all the contracts created by it.  (deprecated)" +
```